src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/PublicationViewModel.cs

```diff
@@ -1,31 +1,50 @@
-#nullable enable
+﻿#nullable enable
 using System;
 using System.Collections.Generic;
 
-namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+
+public record PublicationViewModel
 {
```

Comment:
Could change this to use file namespacing

Reply:
I think this one is already using a file scoped namespace.

Reply:
D'oh, of course! :smile: